package/gluon-web-private-wifi/i18n/de.po

```diff
@@ -13,18 +13,42 @@ msgstr ""
 msgid "8-63 characters"
 msgstr "8-63 Zeichen"
 
+msgid "Disabled"
+msgstr "Deaktiviert"
+
 msgid "Enabled"
 msgstr "Aktiviert"
 
+msgid "Encryption"
+msgstr "Verschlüsselung"
+
 msgid "Key"
 msgstr "Schlüssel"
 
+msgid "Management Frame Protection"
+msgstr ""
+
 msgid "Name (SSID)"
 msgstr "Name (SSID)"
 
+msgid "Optional"
+msgstr ""
+
 msgid "Private WLAN"
 msgstr "Privates WLAN"
 
+msgid "Required"
+msgstr "Aktiviert"
```

Comment:
Erzwingen?

Reply:
Please not. MFP is a feature which can be *activated*, *deactivated* or flagged as *optional*. Erzwingen is not the correct term in this context.